bekeplar/EPIC-Mail
# Ft fetch api get all sent messages 165082404

## What this PR does?
- Adds fetch API capabilities that enable a user to retrieve all their sent messages
## Tasks completed
- writing Js functions to enable the backend incorporate with the front end
- Refactoring some files in the DOM by adding new HTML elements to facilitate fetch
## How should this be manually tested?
On this Pull request navigate to the files changed tab to see the various files changed

Kindly look through fellow developers and advise as possible
@neelxie, @alexxsanya, @Emmanuelaaron, @nanfuka, @Wabuluka, SentongoHabib, @kyakusahmed
